glTF-BinExporter/GlTFUtils.cs

```diff
@@ -119,6 +152,29 @@ public static class GlTFUtils
             return rhinoObjectsRes;
         }
 
+        private static void ExplodeRecursive(InstanceObject instanceObject, Transform instanceTransform, List<RhinoObject> pieces, List<Transform> transforms)
+        {
+            for(int i = 0; i < instanceObject.InstanceDefinition.ObjectCount; i++)
+            {
+                RhinoObject rhinoObject = instanceObject.InstanceDefinition.Object(i);
+
+                pieces.Add(rhinoObject);
```

Comment:
Isn't `InstanceObjects` added to `pieces` as well here? I think this should go in the `else` clause below? They are probably filtered out later, but I wouldn't expect to get InstanceObjects back from this method.

Reply:
Resolved in 68313ba